tools/distrib/python/grpcio_tools/setup.py

```diff
@@ -60,8 +64,9 @@
 GRPC_PYTHON_TOOLS_PACKAGE = 'grpc.tools'
 GRPC_PYTHON_PROTO_RESOURCES_NAME = '_proto'
 
-import protoc_lib_deps
-import grpc_version
+DEFINE_MACROS = (('HAVE_PTHREAD', 1),)
+if "win32" in sys.platform and '64bit' in platform.architecture()[0]:
+  DEFINE_MACROS += (('MS_WIN64', 1),)
```

Comment:
If you mean whether or not the flag needs to be defined somewhere: yes, it's needed.
If you mean, "Isn't this already in the build scripts?" then: this is 'needed' because I want source distributions to eventually be self-contained and buildable on our target systems without requiring that users have intricate knowledge of their compilers and environments in the common case, and I want the flags in the build scripts to eventually be automagical.